### PR TITLE
JSON encode params fields in conn

### PR DESCRIPTION
**Pull request by tonyvanriet**
https://cuatrohq.slack.com/archives/CMZFU35JL/p1623679233093800

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -72,6 +72,7 @@ defmodule Hummingbird.MixProject do
       {:plug, "~> 1.7"},
       {:elixir_uuid, "~> 1.2"},
       {:opencensus_honeycomb, "~> 0.3"},
+      {:jason, "~> 1.2"},
```
Comment (the-mikedavis) [inline mix.exs:75]:
```suggestion
      {:jason, "~> 1.0"},
```

yada yada as wide as possible to make it easy for consumers

**Review by the-mikedavis — APPROVED** [review]
:+1:

File: lib/hummingbird/impl.ex [hunk 2]
```diff
@@ -16,8 +16,29 @@ defmodule Hummingbird.Impl do
...
+  defp encode_fetched_params(params) do
+    case params do
+      %Plug.Conn.Unfetched{} -> params
+      _ -> Jason.encode!(params)
+    end
+  end
+
   def endpoint_name(conn) do
-    inspect(conn.private.phoenix_endpoint) <> ".call/2"
+    inspect(conn.private[:phoenix_endpoint])
```
[the remark this answers is not in the document]
Comment (tonyvanriet) [reply lib/hummingbird/impl.ex:41]: meant to call this out but then never submitted my comments yesterday.
we're not losing anything important here, right @NFIBrokerage/baemax ?

File: mix.exs (hunk 1 above)
Comment (tonyvanriet) [reply mix.exs]: one of these days I'll get this right on my own.
when I do, could I get a log rip please?

File: lib/hummingbird/impl.ex (hunk 2 above)
[the remark this answers is not in the document]
Comment (bjornrud) [reply lib/hummingbird/impl.ex:41]: :ripit:  the waterfall trace display should be the only thing affected that I can think of.

**Pull request merged**